packages/itwinui-css/src/breadcrumbs/breadcrumbs.scss

```diff
@@ -33,20 +34,20 @@
 
 @mixin iui-breadcrumbs-item-overrides {
   > * {
-    padding: 0 $iui-s;
+    padding: 0 var(--iui-size-xs);
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
 
     &:not(.iui-button) {
-      color: var(--iui-text-color);
+      color: var(--iui-color-foreground-2);
     }
   }
 
   > :any-link {
     @include iui-anchor;
     box-sizing: border-box;
-    line-height: $iui-component-height;
+    line-height: var(--_iui-breadcrumbs-height);
```

Comment:
sure we wanna have `line-height` with value of `100%` ?

Reply:
but this is the same as before

Reply:
Well no, previously it was using `iui-component-height` variable values for `line-height`, now you are using param of the mixin value, which is passed as `100%` when breadcrumbs mixin is used in header.

Reply:
Reverted 👍